AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -460,7 +476,10 @@ - (void)_layoutTextView
   [_textKitComponents.layoutManager invalidateLayoutForCharacterRange:NSMakeRange(0, [_textKitComponents.textStorage length]) actualCharacterRange:NULL];
 
   // When you type beyond UITextView's bounds it scrolls you down a line. We need to remain at the top.
-  [_textKitComponents.textView setContentOffset:CGPointZero animated:NO];
+  // [Question - leotumwattana] Is there a good reason why we are forcing content offset to top? Why not
+  // keep at the current offset of the text view?
+  CGPoint offset = _textKitComponents.textView.contentOffset;
+  [_textKitComponents.textView setContentOffset:offset animated:NO];
```

Comment:
Yes, it does look strange, but what I've found is that without this strange `textView.contentOffset = textView.contentOffset` the textView jumps up/down during editing under certain cases when the editableTextNode is scrollable due to `.maximumLinesToDisplay`. 

Whereas the original code (below) will cause the editableTextNode to jump up to top:
```
[_textKitComponents.textView setContentOffset:CGPointZero animated: NO];
```

I'll add an additional comment when I resubmit this as two separate PR.